### ISSUE TITLE
[IG Charter] clarification on the activity of the Marketing TF

### ISSUE THREAD

**Issue by ashimura** [w3c/wot-marketing#217]
To address the comments in [wot Issue 986](https://github.com/w3c/wot/issues/986), we need to add the following clarifications to the [Draft IG Charter](https://w3c.github.io/wot/charters/wot-ig-2021-proposed.html):
* [x] What achievement did we get in the past 2 years for the Marketing TF?
* [x] How does the Marketing TF fit into the scope of the Group?
* [x] #218

**Comment by egekorkan** [comment]
> What achievement did we get in the past 2 years for the Marketing TF?

We have started to list talk about these in the TF meeting. Below is the initial brainstorming:
- Website
- Explainer Video
- Documentation and early work on tutorials
- Documenting available implementations and tools
- Explaining TFs in a more public style

We should have a brief description. The initial proposal from the call of 09.11.2021:
Marketing Task Force works on promoting the W3C Web of Things in different ways such as Web presence, Social Media presence, and more. In the last charter period, we have worked on establishing our website, created an explainer video for the WoT, increased our social media presence, and worked on documenting our work in a more approachable way.
